drush/civicrm.drush.inc

```diff
@@ -86,8 +86,14 @@ function civicrm_drush_command() {
   ];
   $items['civicrm-ext-list'] = [
     'description' => "List of CiviCRM extensions enabled.",
-    'examples' => [
+    'options' => [
+      'status' => 'Filter by extension status (installed, uninstalled, disabled).',
+      'out' => 'Output type: "pretty" (STDOUT) by default, "json" (STDOUT)',
+    ],
+     'examples' => [
```

Comment:
The style-checker doesn't seem to run in this repo but this looks off by one char.